scripts/setup_env.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+sudo apt install git doxygen graphviz -y
```

Comment:
@NeoZhangJianyu @mkbhanda @yinghu5 @chensuyue @intelkevinputnam OPEA doesn't use doxygen?

Reply:
Thank you @dbkinder for continuing to be the doc guardian angle for OPEA!! 
https://doxygen.nl/index.html Doxygen "automates the generation of documentation from source code comments, parsing information about classes, functions, and variables to produce output in formats like HTML and PDF" but I see no signs of its use in our various OPEA subprojects. Just see references to it in opea-projects/docs in scripts/setup_env.sh and scripts/show-versions.py. For that matter I do not see graphviz either. Are these being included in anticipation of use @NeoZhangJianyu ?